src/resolvers/user_management.spec.js

```diff
@@ -83,97 +95,165 @@ describe('currentUser', () => {
-    describe('with valid email/password combination', () => {
-      it('responds with a JWT token', async () => {
-        const data = await request(host, mutation({
-          email: 'test@example.org',
-          password: '1234'
-        }))
-        const token = data.login
+      }`;
+  };
+
+  describe("ask for a `token`", () => {
+    describe("with valid email/password combination", () => {
+      it("responds with a JWT token", async () => {
+        const data = await request(
+          host,
+          mutation({
+            email: "test@example.org",
+            password: "1234"
+          })
+        );
+        const token = data.login;
         jwt.verify(token, process.env.JWT_SECRET, (err, data) => {
-          expect(data.email).toEqual('test@example.org')
-          expect(err).toBeNull()
-        })
-      })
-    })
+          expect(data.email).toEqual("test@example.org");
+          expect(err).toBeNull();
+        });
+      });
+    });
 
-    describe('with a valid email but incorrect password', () => {
+    describe("with a valid email but incorrect password", () => {
       it('responds with "Incorrect email address or password."', async () => {
         await expect(
-          request(host, mutation({
-            email: 'test@example.org',
-            password: 'wrong'
-          }))
-        ).rejects.toThrow('Incorrect email address or password.')
-      })
-    })
-
-    describe('with a non-existing email', () => {
+          request(
+            host,
+            mutation({
+              email: "test@example.org",
+              password: "wrong"
+            })
+          )
+        ).rejects.toThrow("Incorrect email address or password.");
+      });
+    });
+
+    describe("with a non-existing email", () => {
       it('responds with "Incorrect email address or password."', async () => {
         await expect(
-          request(host, mutation({
-            email: 'non-existent@example.org',
-            password: 'wrong'
-          }))
-        ).rejects.toThrow('Incorrect email address or password.')
-      })
-    })
-  })
-})
+          request(
+            host,
+            mutation({
+              email: "non-existent@example.org",
+              password: "wrong"
+            })
+          )
+        ).rejects.toThrow("Incorrect email address or password.");
+      });
+    });
+  });
+});
+
+describe("change password", () => {
+  let headers;
+  let client;
+
+  beforeEach(async () => {
+    headers = await login({ email: "test@example.org", password: "1234" });
+    client = new GraphQLClient(host, { headers });
+  });
+
+  const mutation = params => {
+    const { oldPassword, newPassword } = params;
+    return `
+      mutation {
+        changePassword(oldPassword:"${oldPassword}", newPassword:"${newPassword}")
+      }`;
+  };
+
+  describe("should be authenticated before changing password", () => {
+    it('should throw not "Not Authorised!', async () => {
```

Comment:
```suggestion
    it('throws not "Not Authorised!', async () => {
```
Don't write `should`. Just say, it `does XY` period. It's describing the behaviour in a confident way :wink: 

You could also have a look at the output when you `yarn run test` and check if you like the documentation.

Reply:
Gotcha